### PR TITLE
Fixes #20381 - improved error message during kexec

### PR DESCRIPTION
**Pull request by lzap**
Very useful for kexec debugging.

### CODE REVIEW

**Comment by lzap** [comment]
Ping anyone @orrabin this is blocker bug, kexec does not work at all. [test]

**Comment by lzap** [comment]
Oh wait not a blocker, sorry. Normal priority :-)

**Comment by lzap** [comment]
Just a core JS failure, unrelated. Not a blocker but nice to have in so setting important flag as well.

File: app/models/host/managed_extensions.rb [hunk 1]
```diff
@@ -50,6 +53,7 @@ def setKexec
     json = JSON.parse(unattended_render(template))
     ::Foreman::Exception.new(N_("Kernel kexec URL is invalid: '%s'"), json['kernel']) unless json['kernel'] =~ /\Ahttp.+\Z/
     ::Foreman::Exception.new(N_("Init RAM kexec URL is invalid: '%s'"), json['initrd']) unless json['initrd'] =~ /\Ahttp.+\Z/
+    logger.info "KEXEC payload: #{json}"
```
Comment (lzap) [inline app/models/host/managed_extensions.rb]: Shall I remove this one? I did not notice we have this in logs already.

Comment (orrabin) [reply app/models/host/managed_extensions.rb]: Yes. Please also change the commit message to fit the change that is left.

**Comment by lzap** [comment]
Done!

**Comment by lzap** [comment]
This is a quick merge @dLobatog or @timogoebel or @orrabin just an extra log message.

**Comment by lzap** [comment]
Ok I am going to merge this trivial patch to trigger develop build, testing CI.

**Pull request merged**